package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "bookstore_react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-browser-router": "^2.1.2",
```

Comment:
✅ The project should be initialized using create-react-app 👍🏻

Reply:
Hey, i initialized the app with create-react-app